.github/workflows/windows-java.yml

```diff
@@ -0,0 +1,113 @@
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install packages
+        uses: msys2/setup-msys2@v2
+        with:
+          update: true
+          install: autoconf automake libtool make mingw-w64-x86_64-ncurses mingw-w64-x86_64-libxml2 mingw-w64-x86_64-cjson mingw-w64-x86_64-db mingw-w64-x86_64-gmp libdb-devel mingw-w64-x86_64-gcc flex bison gmp-devel help2man texinfo gettext-devel
+
+      - name: Set git user
+        run: |
+          git config --global user.name github-actions
+          git config --global user.email github-actions-bot@users.noreply.github.com
+
+      - name: bootstrap
+        run: |
+          ./autogen.sh
+          autoconf
+          autoreconf --install --force
+        shell: msys2 {0}
+
+      - name: Build environment setup
+        run: |
+          mkdir _build
+          echo "JAVA_HOME=$JAVA_HOME"
+          echo "export PATH=\"$(cygpath -u "$JAVA_HOME")/bin:\$PATH\"" >> ~/.bash_profile
+        shell: msys2 {0}
+
+      - name: configure
+        run: |
+          cd _build
+          echo "$PATH"
+          java_home="$(cygpath -u "$JAVA_HOME")"
+          # thanks @revelator & @GitMensch ;-)
+          # (cf https://github.com/msys2/MINGW-packages/discussions/8056)
+          lt_cv_deplibs_check_method=pass_all \
+          ../configure --enable-cobc-internal-checks \
+                       --enable-hardening \
+                       --prefix /opt/cobol/gnucobol \
+                       --exec-prefix /opt/cobol/gnucobol \
+                       --with-java \
+                       JAVA_HOME="$java_home" \
+                       JAVA="$java_home/bin/java" \
+                       JAVAC="$java_home/bin/javac"
+        shell: msys2 {0}
+
+      - name: Upload config.log
+        uses: actions/upload-artifact@v4
+        # if: failure()
+        with:
+          name: config-${{ matrix.os }}.log
+          path: _build/config.log
+
+      - name: make
+        run: |
+          make -C _build --jobs=$(($(nproc)+1))
+        shell: msys2 {0}
+
+      - name: install
+        run: |
+          make -C _build install
+          find /opt/cobol > _build/install.log
+        shell: msys2 {0}
+
+      - name: Upload install.log
+        uses: actions/upload-artifact@v4
+        with:
+          name: install-${{ matrix.os }}.log
+          path: _build/install.log
+
+      - name: Java runtime setup
+        shell: msys2 {0}
+        run: |
+          # Be sure to have `jvm.dll` in path for tests with JNI below
+          jvmdir="$(dirname $(find "$JAVA_HOME" -name jvm.dll))"
+          echo "export PATH=\"$(cygpath -u "$jvmdir"):\$PATH\"" >> ~/.bash_profile
+
+      - name: check
+        continue-on-error: true
+        run: |
+          sed '/AT_SETUP(\[ACCEPT OMITTED (SCREEN)\])/a \
+                AT_SKIP_IF(\[true\])' \
+              -i tests/testsuite.src/run_accept.at
```

Comment:
Isn't setting TERM during `make check` enough? Why do we need that test to be skipped?

Reply:
No idea (I'll see what happens if I remove these lined).

Reply:
Well… the tests hang. What's the proper way to set TERM in here?

Reply:
just inline it, something like `TERM=vt100 make check` (vt100+xterm are the two I've seen most often in CI definitions, but somehow TERM=dumb seems indeed to be better).

Reply:
That hangs with `TERM=dumb`. I'll try with `TERM=vt100`. Not sure that's what's actually seen by the program though (*cf* https://sourceforge.net/p/msys2/tickets/241/ — I don't know when `/etc/profile` is sourced though).

Reply:
Still hangs. I guess for now I'll restore the above lines.